react/Accordion/index.jsx

```diff
@@ -0,0 +1,85 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import cx from 'classnames'
+import Icon from '../Icon'
+import styles from './styles.styl'
+
+class AccordionItem extends Component {
+  constructor(props) {
+    super(props)
+    this.onClick = this.onClick.bind(this)
+    this.state = { selected: false }
+  }
+
+  render() {
+    const { children, label, className } = this.props
+
+    const { selected } = this.state
+
+    const bodyName = label.replace(/\s+/g, '')
```

Comment:
Move this instruction in a named function to explain what it does.
As I understand RegExp, it removes every space characters.

```js
const removeSpaceCharacters = str => str.replace(/\s+/g, '')
```